configure.ac

```diff
@@ -53,6 +53,11 @@ FUZZ_CPPFLAGS="${FUZZ_CPPFLAGS:--DASS_FUZZMODE=0}"
 # Checks for flags support
 AX_APPEND_COMPILE_FLAGS([-fno-math-errno])
 
+# Enable some warnings
+AX_APPEND_COMPILE_FLAGS([-Wall -Wextra -Wno-sign-compare -Wno-unused-parameter])
+AX_APPEND_COMPILE_FLAGS([-Werror-implicit-function-declaration -Wstrict-prototypes])
+AX_APPEND_COMPILE_FLAGS([-Wpointer-arith -Wredundant-decls -Wno-missing-field-initializers])
+
```

Comment:
Was wondering why this is split into three macro calls, but the need for escaping the initial and final newline might be surprising and unintuitive with C-like syntax in mind
```autoconf
AX_APPEND_COMPILE_FLAGS([ \
    -Wall -Wextra -Wno-sign-compare -Wno-unused-parameter \
    -Werror-implicit-function-declaration -Wstrict-prototypes \
    -Wpointer-arith -Wredundant-decls -Wno-missing-field-initializers \
])
```

Feel free to keep the current version or use the above if you think it’s clear enough

Reply:
Sure, fair enough.